assets/js/components/Household/create-household.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Will users be able to select roommates on this page? Totally fine with that coming in a future PR, just curious if that's the plan

Reply:
Ya, I believe we will add that here, likely as a search box that allows for the searching of existing users. And then likely we will need an invite table, or perhaps we just take a shortcut and add users directly if they are added to household.